object-group-game/Inventory.cs

```diff
@@ -4,6 +4,23 @@
 
 namespace object_group_game {
 	class Inventory {
-		public List<Item> Items { get; set; }
+		private List<Item> _items;
+		public List<Item> Items {
```

Comment:
Do we really need cloning for this? (Just asking, I'm not sure about how you envision this)
(C# please)

Reply:
I think list of items should not be manipulated outside the Inventory class. I believe returning a clone is the best solution for that. Now, when I think about it, auto-generated get method should be private and return the actual reference to the list for the class methods to be able to manipulate it. An additional method should be created to get it from outside the class. It sounds right in my mind, would appreciate a second opinion.